Scripts/update-translations.rb

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env ruby
-# encoding: utf-8
```

Comment:
RuboCop removed this because UTF-8 is the default encoding in [Ruby 2+](https://www.ruby-lang.org/en/news/2013/02/24/ruby-2-0-0-p0-is-released/).